src/App.js

```diff
@@ -1,5 +1,131 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+
+const CAR_NAME_INPUT_GUIDE = "ex) 'pobi,jun' 같은 형식으로 입력해주세요.";
+const GAME_COUNT_INPUT_GUIDE = "ex) '1' 같은 양수 정수로 입력해주세요.";
+
+const throwNewError = (condition, errorMessage) => {
+  if (condition) {
+    throw new Error(errorMessage);
+  }
+};
```

Comment:
특정 조건에서 에러를 던져주는 역할을 따로 분리한 점이 너무 좋았습니다!
내부에서 에러메시지 폼을 작성해주시면 아래에서 반복적으로 `[ERROR]`을 포함하지 않고 메시지 내용만 작성하실 수 있습니다.

```suggestion
const throwNewError = (condition, errorMessage) => {
  if (condition) {
    throw new Error(`[ERROR] ${errorMessage}`);
  }
};
```
특정 예외조건에 대한 에러처리라면 커스텀 에러를 만들어서 throw 하는 방법도 있습니다!
- [관련링크 : 커스텀 에러와 에러 확장](https://ko.javascript.info/custom-errors)

Reply:
prefix를 따로 빼면 메시지 내용만 작성할 수 있어서 더 좋겠네요!! 😊 
좋은 피드백 감사합니다!!